src/Rule.php

```diff
@@ -2949,12 +2949,19 @@ public function getAllActions()
 
     public function getActions()
     {
-        return [
-            '_stop_rules_processing' => [
-                'name' => __('Skip remaining rules'),
-                'type' => 'yesonly',
-            ]
-        ];
+        $actions = [];
+        $collection_class = static::getCollectionClassName();
+        if (class_exists($collection_class)) {
```

Comment:
A rule should always be attached to a collection class. I guess the `getCollectionClassName` method returns inconsistent result for `LevelAgreementLevel`.
As `LevelAgreementLevel` extends `RuleTicket`, the corresponding collection should probably be `RuleTicketCollection`, that uses the default `false` value for its `stop_on_first_match` property, and so the `_stop_rules_processing` should be available.

IMHO, the `getCollectionClassName()` should probably be changed to loop on `get_parent_class()` until it found an existing class that extends `RuleCollection`.